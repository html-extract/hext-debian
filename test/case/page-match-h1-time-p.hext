<h1 class:h1_class />
<?time class:time_class />
<p class:p_class />
