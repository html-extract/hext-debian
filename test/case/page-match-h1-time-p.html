<div>
  <h1 class="heading1">a</h1>
  <time  class="time1">a</time>
  <p        class="p1">a</p>
  <h1 class="heading2">a</h1>
  <time  class="time2">a</time>
  <p        class="p2">a</p>
  <h1 class="heading3">a</h1>
  <time  class="time3">a</time>
  <p        class="p3">a</p>
  <h1 class="heading4">a</h1>
  <p        class="p4">a</p>
  <h1 class="heading5">a</h1>
  <time  class="time5">a</time>
  <p        class="p5">a</p>
</div>
