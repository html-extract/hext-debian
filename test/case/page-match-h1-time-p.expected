{"h1_class":"heading1","p_class":"p1","time_class":"time1"}
{"h1_class":"heading2","p_class":"p2","time_class":"time2"}
{"h1_class":"heading3","p_class":"p3","time_class":"time3"}
{"h1_class":"heading4","p_class":"p4"}
{"h1_class":"heading5","p_class":"p5","time_class":"time5"}
